common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -15,10 +15,21 @@
 		culture_group = culture_group:titanforged_group
 		culture_group = culture_group:troll_group
 		culture_group = culture_group:undead_group
-		scope:culture = culture:dark_iron
```

Comment:
Created even more bugs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110964969-32176500-836d-11eb-9ebc-5d67ba9c8a55.png)

</details>